libraries/AP_SerialManager/AP_SerialManager.h

```diff
@@ -151,14 +157,15 @@ class AP_SerialManager {
         SerialProtocol_MSP = 32,
         SerialProtocol_DJI_FPV = 33,
         SerialProtocol_Sagetech = 34,
+        SerialProtocol_SmartAudio=35,
```

Comment:
Please observe whitespace conventions in our coding standards. Here and elsewhere: https://ardupilot.org/dev/docs/style-guide.html there is a tool you can use to get you started on the new code.